website/docs/r/content_library.html.markdown

```diff
@@ -3,16 +3,30 @@ layout: "vcfa"
 page_title: "VMware Cloud Foundation Automation: vcfa_content_library"
 sidebar_current: "docs-vcfa-resource-content-library"
 description: |-
-  Provides a VMware Cloud Foundation Automation Content Library resource. This can be used to manage Content Libraries.
+  Provides a resource to manage Content Libraries in VMware Cloud Foundation Automation. It can be used to upload
+  Content Library Items such as ISO files, OVAs and OVFs.
 ---
 
 # vcfa\_content\_library
 
-Provides a VMware Cloud Foundation Automation Content Library resource. This can be used to manage Content Libraries.
+Provides a resource to manage Content Libraries in VMware Cloud Foundation Automation. It can be used to upload
```

Comment:
Maybe I should not use the word "upload" here, could be misleading?
```suggestion
Provides a resource to manage Content Libraries in VMware Cloud Foundation Automation. It can be used to host
```
As the one that uploads is the Content Library Item. The hyperlink sort of guides into that conclusion, but still...